translations/bn/LC_MESSAGES/messages.po

```diff
@@ -2077,18 +2089,6 @@ msgstr "Subscribe to the newsletter"
 msgid "surname"
 msgstr "Username"
 
-#, fuzzy
```

Comment:
Tagging @hasan-sh for this one. Are these tags still in use or are they deleted correctly?

Reply:
I made a quick search on main, and these tags are not included in any HTML or python file, so maybe it was correct?

Reply:
I don't get why these are removed...

Reply:
I just started a discusion in Discord, I think I understand why, but this is in any case not a problem with the PR @Annelein